docker/fedora/Dockerfile

```
@@ -0,0 +1,32 @@
+ARG BASE_IMAGE=fedora:42
+FROM $BASE_IMAGE AS runtime
+
+ARG ALUMET_VERSION=0.8
+LABEL org.opencontainers.image.authors="maintainers@alumet.dev" \
+	org.opencontainers.image.documentation="https://alumet-dev.github.io/user-book" \
+	org.opencontainers.image.source="https://github.com/alumet-dev/alumet" \
+	org.opencontainers.image.version="$ALUMET_VERSION" \
+	org.opencontainers.image.vendor="alumet-dev" \
+	org.opencontainers.image.licenses="EUPL-1.2" \
+	org.opencontainers.image.title="alumet-agent based on Fedora OS" \
+	org.opencontainers.image.description="Alumet is a modular tool that measures energy consumption and performance metrics. "
+
+ARG ARTIFACT_FILE=alumet-agent-0.8.0-1.fedora.42.x86_64.rpm
```

```suggestion
ARG ARTIFACT_FILE=alumet-agent-0.8.0-1.fc42.x86_64.rpm
```
Should be `fc42` because that's the name produced by the RPM workflow. See for example https://github.com/alumet-dev/packaging/actions/runs/14036275524/job/39294934862

---

Sorry, one change I've just noticed! We could argue whether we want to change the name here or in the RPM workflow, but it needs to be changed somewhere :)